src/platform/packages/shared/kbn-unified-data-table/src/components/row_height_settings.tsx

```diff
@@ -18,41 +24,30 @@ export enum RowHeightMode {
 }
 export interface RowHeightSettingsProps {
   rowHeight?: RowHeightMode;
-  rowHeightLines?: number;
   maxRowHeight?: number;
   label: string;
-  compressed?: boolean;
   onChangeRowHeight: (newHeightMode: RowHeightMode | undefined) => void;
   onChangeRowHeightLines: (newRowHeightLines: number) => void;
   'data-test-subj'?: string;
+  lineCountInput: number;
 }
 
 const idPrefix = htmlIdGenerator()();
 
 export function RowHeightSettings({
   label,
   rowHeight,
-  rowHeightLines,
-  compressed,
   onChangeRowHeight,
   onChangeRowHeightLines,
   maxRowHeight,
   ['data-test-subj']: dataTestSubj,
+  lineCountInput,
```

Comment:
Nit: if this replaces the `rowHeightLines` prop, I'd recommend updating the `onChangeRowHeightLines` props to `onChangeLineCountInput` too for consistency.

Reply:
Good idea, will do!